pkg/kubelet/pleg/generic.go

```diff
@@ -126,9 +141,33 @@ func (g *GenericPLEG) Watch() chan *PodLifecycleEvent {
 	return g.eventChannel
 }
 
+func (g *GenericPLEG) Relist() {
+	g.relist()
```

Comment:
I don't see much reason in making this immediately call a private function of the same name. I am not seeing anywhere `relist()` is called a second time, and even if it was, given `relist()` is all that happens, `Relist()` could just as easily be called

Reply:
Moved the functionality within `relist()` to `Relist()` directly in order to avoid duplication.